### PR TITLE
feat(cli): only use ansi logging on tty

### PR DESCRIPTION
**Pull request by adamreese**

### CODE REVIEW

File: bin/client/main.rs [hunk 1]
```diff
@@ -75,7 +75,9 @@ impl TokenManager for PickYourAuth {
 async fn run() -> std::result::Result<(), ClientError> {
     let opts = opts::Opts::parse();
     // TODO: Allow log level setting outside of RUST_LOG (this is easier with this subscriber)
-    tracing_subscriber::fmt::init();
+    tracing_subscriber::fmt()
+        .with_ansi(atty::is(atty::Stream::Stderr))
```
Comment (thomastaylor312) [inline bin/client/main.rs:79]: I just realized we didn’t change the client to use stderr for logging. Should just be able to copy paste from server

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
One minor change needed

File: bin/client/main.rs (hunk 1 above)
Comment (technosophos) [reply bin/client/main.rs:79]: You mean essentially this?

https://github.com/deislabs/bindle/blob/main/bin/server.rs#L157-L160

Comment (thomastaylor312) [reply bin/client/main.rs:79]: Yep, exactly!

Comment (adamreese) [reply bin/client/main.rs:81]: @thomastaylor312 Can you check to make sure I understood correctly?

Comment (thomastaylor312) [reply bin/client/main.rs:81]: Yep, that is correct!

**Review by thomastaylor312 — APPROVED** [review]

**Pull request merged**